code/modules/hydroponics/plant_genes.dm

```diff
@@ -355,8 +354,8 @@
 
 			// The secret of potato supercells!
 			var/datum/plant_gene/trait/cell_charge/CG = G.seed.get_gene(/datum/plant_gene/trait/cell_charge)
-			if(CG) // 10x charge for deafult cell charge gene - 20 000 with 100 potency.
-				pocell.maxcharge *= CG.rate*1000
+			if(CG) // Cell charge increases amount of power but not higher than bluespace
```

Comment:
give actual numbers for the resulting maxcharge

Reply:
See above.